combineRooms/combineRooms.py

```diff
@@ -0,0 +1,422 @@
+        """
+        is_public = await self._toggle_public_combine(ctx.guild)
+        await self._update_combine_permissions(ctx.guild)
+
+        public_str = "public" if is_public else "private"
+        response = "Combines are now **{0}**.".format(public_str)
+        await ctx.send(response)
+
+    @commands.command(aliases=["checkCombinePublicity", "ccp", "combineStatus", "checkCombineStatus", "ccs"])
+    @commands.guild_only()
+    @checks.admin_or_permissions(manage_guild=True)
+    async def combinePublicity(self, ctx):
+        """
+        Gets the current status (public/private) of the combines.
+        If combines are **Public**, any member may participate.
+        If combines are **Private**, only members with the "League" role may particpate.
+        """
+        public_str = "public" if await self._is_public_combine(ctx.guild) else "private"
+        response = "Combines are currently **{0}**.".format(public_str)
+        await ctx.send(response)
+
+    @commands.command(aliases=["getAcronym"])
+    @commands.guild_only()
+    @checks.admin_or_permissions(manage_guild=True)
+    async def acronym(self, ctx):
+        """
+        Gets the acronym registered for combines. (Default: RSC)
+        """
+        acronym = await self._get_acronym(ctx.guild)
+        await ctx.send("The acronym registered for the combines cog is **{0}**.".format(acronym))
+
+    @commands.command()
+    @commands.guild_only()
+    @checks.admin_or_permissions(manage_guild=True)
+    async def setAcronym(self, ctx, new_acronym: str):
+        """
+        Sets the server acronym used in the combines category. (Default: RSC)
+        This is mostly used in #combine-details message
+        """
+        await self._save_acronym(ctx.guild, new_acronym)
+        await ctx.send("The acronym has been registered as **{0}**.".format(new_acronym))
+
+    @commands.Cog.listener("on_voice_state_update")
+    async def on_voice_state_update(self, member, before, after):
+        combines_ongoing = await self._combines_category(member.guild)
+
+        # ignore when combines are not ongoing, or when voice activity is within the same room
+        if not combines_ongoing or before.channel == after.channel:
+            return
+        
+        # Room joined:
+        await self._member_joins_voice(member, after.channel)
+        
+        # Room left:
+        await self._member_leaves_voice(member, before.channel) # TODO: consider disconnected case #@me what does that even mean? this structure should cover everything
+
+    async def _start_combines(self, ctx):
+        # Creates combines category and rooms for each tier
+        combines_category = await self._add_combines_category(ctx, "Combine Rooms")
+        await self._save_combine_category(ctx.guild, combines_category)
+
+        if combines_category:
+            await self._add_combines_info_channel(ctx.guild, "Combines Details")
```

Comment:
I'd just pass in the category that was just created into this method rather than passing in the guild and then getting the category from there. No point in creating another copy of the same variable.

Reply:
Same for line 175.

Reply:
Done